index.bs

```diff
@@ -5916,6 +5917,74 @@ When [=matching capabilities=], the extension-specific steps to match `"webauthn
         the match is unsuccessful.
     2. Otherwise, the match is successful.
 
+### <dfn>Authenticator Extension Capabilities</dfn> ### {#sctn-authenticator-extension-capabilities}
+
+Additionally, [=extension capabilities=] are defined for every [=authenticator extension=] defined in this specification:
+
+<figure id="table-virtualAuthenticatorsExtensionsWebdriverCapability" class="table">
+    <table class="data">
+        <thead>
+            <tr>
+                <th>Capability</th>
+                <th>Key</th>
+                <th>Value Type</th>
+                <th>Description</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td>FIDO AppID Extension Support</td>
+                <td>`"webauthn:appid"`</td>
```

Comment:
How about using a second namespace level for extensions, like `webauthn:extension:appid` or `webauthn:ext:appid`? I'm thinking capabilities like `webauthn:pin-userVerification` or something like that might be useful at some point, maybe for features added in the future, and it seems appropriate to separate the extensions from other features.